terraform/environments/electronic-monitoring-data/transfer_server_capita.tf

```diff
@@ -89,11 +89,20 @@ resource "aws_transfer_server" "capita_transfer_server" {
   security_policy_name = "TransferSecurityPolicy-2023-05"
 
   pre_authentication_login_banner = "Hello there"
-  logging_role                    = aws_iam_role.test_transfer_user_iam_role.arn
+  
+  workflow_details {
+    on_upload {
+      workflow_id = aws_transfer_workflow.transfer_capita_to_store.id
+      execution_role = aws_iam_role.capita_transfer_workflow_iam_role.arn
+    }
+  }
```

Comment:
@modernisation-platform This is the code that's triggered needing a review from you